src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -110,6 +110,7 @@ class misc_commandscript : public CommandScript
             { "repairitems",      rbac::RBAC_PERM_COMMAND_REPAIRITEMS,       true, &HandleRepairitemsCommand,      "" },
             { "respawn",          rbac::RBAC_PERM_COMMAND_RESPAWN,          false, &HandleRespawnCommand,          "" },
             { "revive",           rbac::RBAC_PERM_COMMAND_REVIVE,            true, &HandleReviveCommand,           "" },
+            { "reviveall",        rbac::RBAC_PERM_COMMAND_REVIVEALL,         true, &HandleReviveAllCommand,        "" },
             { "saveall",          rbac::RBAC_PERM_COMMAND_SAVEALL,           true, &HandleSaveAllCommand,          "" },
```

Comment:
imho better a subcommand to group.

Reply:
Could you elaborate?

Do you mean as not to create a new `RBAC_PERM_COMMAND_REVIVEALL` permission but just use existing `RBAC_PERM_COMMAND_REVIVE`?

Reply:
i mean instead reviveall put it into cs_group and name it group revive.

Reply:
Oh sorry, I'm new to codebase. I see what you mean. Pushed